# HG changeset patch
# User Rik <rik@octave.org>
# Date 1550365175 28800
#      Sat Feb 16 16:59:35 2019 -0800
# Branch stable
# Node ID 3a10bdbe6839a5548e0d44bf640f93f001f46405
# Parent  fb920b730931f765e75c9959d61a7e987615fc57
Use "-loose" option for printing by default until bug resolved (bug #48689).

* NEWS: Remove paragraph about "-tight" being default.

* print.m: Change default in documentation to "-tight".

* __print_parse_opts__.m: Change arg_st.tight default to false.

diff --git a/NEWS b/NEWS
--- a/NEWS
+++ b/NEWS
@@ -153,22 +153,16 @@ Summary of important user-visible change
  ** Printing using the -dtiff output device will now create compressed
     images using LZW compression.  This change was made for Matlab
     compatibility.  To produce uncompressed images use the -dtiffn
     device.
 
  ** A new printing device is available, -ddumb, which produces ASCII art
     for plots.  This device is only available with the gnuplot toolkit.
 
- ** Printing to EPS files now uses a tight bounding box ("-tight"
-    argument to print) by default.  This makes more sense for EPS
-    files which are normally embedded within other documents, and is
-    Matlab compatible.  If necessary use the "-loose" option to
-    reproduce figures as they appeared in previous versions of Octave.
-
  ** Specifying legend position with a numeric argument is deprecated and
     will be removed in Octave 7.0.  Use a string argument instead.
 
  ** It is now possible to use files and folders containing Unicode
     characters in Windows.
 
  ** The environment variable used by mkoctfile for linker flags is now
     LDFLAGS rather than LFLAGS.  LFLAGS is deprecated, and a warning
diff --git a/scripts/plot/util/print.m b/scripts/plot/util/print.m
--- a/scripts/plot/util/print.m
+++ b/scripts/plot/util/print.m
@@ -184,19 +184,19 @@
 ## (unusually (SVG))@.  For all vector formats, including PDF, PS, and EPS, the
 ## plot size is specified in points.  This option is equivalent to changing the
 ## width and height of the output by setting the figure property
 ## @code{paperposition(3:4)}.  When using the command form of the print
 ## function you must quote the @var{xsize},@var{ysize} option to prevent the
 ## Octave interpreter from recognizing the embedded comma (',').  For example,
 ## by writing @w{"-S640,480"}.
 ##
-## @item  -tight
-## @itemx -loose
-##   Force a tight or loose bounding box for EPS files.  The default is tight.
+## @item  -loose
+## @itemx -tight
+##   Force a tight or loose bounding box for EPS files.  The default is loose.
 ##
 ## @item -@var{preview}
 ##   Add a preview to EPS files.  Supported formats are:
 ##
 ##   @table @code
 ##   @item -interchange
 ##     Provide an interchange preview.
 ##
diff --git a/scripts/plot/util/private/__print_parse_opts__.m b/scripts/plot/util/private/__print_parse_opts__.m
--- a/scripts/plot/util/private/__print_parse_opts__.m
+++ b/scripts/plot/util/private/__print_parse_opts__.m
@@ -59,17 +59,17 @@ function arg_st = __print_parse_opts__ (
   arg_st.printer = "";
   arg_st.renderer = "auto";
   arg_st.resize_flag = "";
   arg_st.rgb_output = false;
   arg_st.send_to_printer = false;
   arg_st.special_flag = "textnormal";
   arg_st.svgconvert = false;
   arg_st.svgconvert_binary = __quote_path__ (__svgconv_binary__ ());
-  arg_st.tight = true;
+  arg_st.tight = false;
   arg_st.use_color = 0; # 0=default, -1=mono, +1=color
 
   if (isunix ())
     arg_st.lpr_options = "-l";
   elseif (ispc ())
     arg_st.lpr_options = "-o l";
   else
     arg_st.lpr_options = "";
